openmetadata-service/src/main/java/org/openmetadata/service/util/EmailUtil.java

```diff
@@ -173,7 +173,7 @@ public Email buildEmailWithDefaultSender(EmailRequest request) {
         && request.getSubject() != null
         && !request.getSubject().equals("")) {
       // Sender Details
-      emailBuilder.from(defaultSmtpSettings.getUsername());
+      emailBuilder.from(defaultSmtpSettings.getSenderMail());
 
```

Comment:
There is a lot of code of this format in this class:
`pair.getName() != null && !pair.getName().equals("")`

You can use `CommonUtil.nullOrEmpty()` method